commands/pull_request.go

```diff
@@ -186,6 +188,14 @@ func pullRequest(cmd *Command, args *Args) {
 		utils.Check(err)
 	}
 
+	if flagPullRequestEditor {
+		editor, err = github.NewEditor("PULLREQ", "pull request", title + "\n\n" + body)
```

Comment:
Why does this repeat the above logic? Couldn't we just have the above conditional branch happen unconditionally if `--edit` was used?